hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestDeleteFromTable.scala

```diff
@@ -80,4 +82,36 @@ class TestDeleteFromTable extends HoodieSparkSqlTestBase {
       }
     })
   }
+
+  test("Test deleting from empty table") {
+    withRecordType()(withTempDir { tmp =>
+      Seq("cow", "mor").foreach { tableType =>
+        val tableName = generateTableName
+        spark.sql(
+          s"""
+             |CREATE TABLE $tableName (
+             |  id int,
+             |  dt string,
+             |  name string,
+             |  price double,
+             |  ts long
+             |) USING hudi
+             | tblproperties (
+             |    primaryKey = 'id',
+             |    tableType = '$tableType'
+             | )
+             | PARTITIONED BY (dt)
+             | LOCATION '${tmp.getCanonicalPath}'
+         """.stripMargin)
+
+
+        // Delete single row
+        spark.sql(s"DELETE FROM $tableName WHERE id = 1")
```

Comment:
What I think should be fixed is why does this op can not create a instance file with schema. I think fix it will better

Reply:
Agree.  @lvyanquan could you check why the schema is not added to the commit metadata for DELETE operation?

Reply:
Thanks for your suggestion😊. The code has been readjusted to address this issue.